console/src/utils/localStorage.ts

```diff
@@ -0,0 +1,79 @@
+import globals from '../Globals';
+
+export const setLSItem = (key: string, data: string) => {
+  window.localStorage.setItem(key, data);
+};
+
+export const getLSItem = (key: string) => {
+  return window.localStorage.getItem(key);
+};
+
+export const removeLSItem = (key: string) => {
+  if (getLSItem(key)) {
+    window.localStorage.removeItem(key);
+  }
+};
+
+type expiryValue = {
+  value: string;
+  expiry: number;
+};
+
+export const setLSItemWithExpiry = (key: string, data: string, ttl: number) => {
+  const now = new Date();
+
+  const item: expiryValue = {
+    value: data,
+    expiry: now.getTime() + ttl,
+  };
+
+  setLSItem(key, JSON.stringify(item));
+};
+
+export const getItemWithExpiry = (key: string) => {
+  const lsValue = getLSItem(key);
+  if (!lsValue) {
+    return null;
+  }
+  const item: expiryValue = JSON.parse(lsValue);
+  const now = new Date();
+
+  if (now.getTime() > item.expiry) {
+    window.localStorage.removeItem(key);
+    return null;
+  }
+
+  return item.value;
+};
+
+// NOTE: use with extreme caution
+export const clearLS = () => {
+  window.localStorage.clear();
+};
+
+export const listLSKeys = () => {
+  return Object.keys(window.localStorage);
+};
+
+// This is the list of the localStorage keys that are being used.
+export const lsKeys = {
```

Comment:
I think considering we have a set of constants here, it will be good if we use `LS_KEYS`.
whenever we use it in the component we can quickly Identify the global constants. 
:see_no_evil: do as you wish

Reply:
I have refactored this